app/client/src/components/pages/MonitoringReport.js

```diff
@@ -1485,7 +1502,7 @@ function CharacteristicsTableSection({
             autoResetFilters={false}
             autoResetSortBy={false}
             data={tableData}
-            defaultSort="name"
+            defaultSort="selected"
```

Comment:
```suggestion
            defaultSort="measurementCount"
```

I know the ticket says to sort by selected, but I think we really need to sort by measurment count per Kiki's notes on 9/14.

Reply:
Good call, I missed that. One reservation I had with sorting by measurement count is that the selected characteristic could still get lost if there are many with the same count. However, it looks like there is multi-sorting, so I can look into that.